service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/CreateAzureDatabaseStep.java

```diff
@@ -146,6 +153,30 @@ private Optional<String> waitForCreateDatabaseContainer(CoreV1Api aksApi, String
         Duration.ofSeconds(10));
   }
 
+  private void logPodLogs(CoreV1Api aksApi, String podName, Integer tailLines) {
+    try {
+      var pod = aksApi.readNamespacedPod(podName, aksNamespace, null);
+      logger.info("Pod {} final status: {}", podName, pod.getStatus());
+
+      try (InputStream logs =
+          new PodLogs(aksApi.getApiClient())
+              .streamNamespacedPodLog(
+                  aksNamespace,
+                  podName,
+                  CREATE_DB_CONTAINER,
+                  null,
+                  Optional.ofNullable(tailLines).orElse(1000),
+                  false)) {
+
+        new BufferedReader(new InputStreamReader(logs))
+            .lines()
+            .forEach(line -> logger.info("Pod {} log line: {}", podName, line));
```

Comment:
Might be helpful to have a more greppable prefix for these embedded loglines so they can be more easily distinguished from the status loglines sprinkled elsewhere.